applications/serial_lte_modem/Kconfig

```
@@ -8,6 +8,11 @@ source "Kconfig.zephyr"
 
 menu "Nordic Serial LTE Modem"
 
+config SLM_CUSTOMIZED
+	bool "Flag of customizing SLM"
+	help
+	  Enable this flag to add customized logic.
```

```suggestion
	  Enable this flag to include customized logic.
```
This flag by itself won't add anything, it will only include additional functionality that must be added to the code.